kernelci.org/content/en/labs/_index.md

```diff
@@ -13,4 +13,6 @@ If you are interested it just sending the results of test runs in your infrastru
 There are a few ways labs can be connected to KernelCI. The documentation for connecting labs in the new architecture is evolving as we add labs.
 
 * [Connecting a LAVA lab](../api_pipeline/pipeline/connecting-lab)
-* Writing your own bride/runtime using Maestro APIs (TBD)
\ No newline at end of file
+* Writing your own bridge/runtime using Maestro APIs (doc TBD, see [this GitHub issue](https://github.com/kernelci/kernelci-project/issues/349#issuecomment-2097641023))
+
+We know that the documentation above may not answer all your questions. We working to improve it. We ask you to reach out to our mailing list at [kernelci@lists.linux.dev](mailto:kernelci@lists.linux.dev)  with questions and feedback. We are eager to hear from you!
```

Comment:
The same comment as [above](https://github.com/kernelci/kernelci-project/pull/406/files#r1666310359).